### PR TITLE
Delete install directory on successful (auto) installation

### LINKED ISSUE #204: if  PS_INSTALL_AUTO=1 the install folder should be removed
In a "Docker"  style , we should delete the install directory after Autoinstall succeeds.

Right now users have to remove it manually to access the admin panel.

### PR DESCRIPTION
**Pull request by sergioregueira**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description? | Deletes the `install` directory when automatic installation was successful.
| Type         | improvement
| BC breaks    | no
| Deprecations | no
| Fixed ticket | Fixes #204.
| How to test  | See detailed info below.

**Scenarios to test:**
Start Docker container with env var `PS_INSTALL_AUTO=1` and:
* Proper DB params (successful installation): `install/` directory must be deleted.
* Invalid DB config (installation error): `install/` directory must NOT be deleted.

Additionally, verify it supports custom install directory (use `PS_FOLDER_INSTALL` env var).

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]
@Quetzacoalt91 Wdyt?

**Review by Quetzacoalt91 — APPROVED** [review]

File: base/config_files/docker_run.sh [hunk 1]
```diff
@@ -91,6 +91,9 @@ if [ ! -f ./config/settings.inc.php ] && [ ! -f ./install.lock ]; then
 
         if [ $? -ne 0 ]; then
             echo 'warning: PrestaShop installation failed.'
+        else
+            echo "\n* Removing install folder..."
+            rm -r /var/www/html/$PS_FOLDER_INSTALL/
```
Comment (jolelievre) [inline base/config_files/docker_run.sh:96]: @Quetzacoalt91 @PierreRambaud I know you already approved this, but removing the whole folder might be troublesome in some cases (like if you want to debug the installation in a docker for instance 😅 )
So I would suggest an alternative to just rename it, this way it's easy to fallback on the still existing folder
```suggestion
            mv /var/www/html/$PS_FOLDER_INSTALL/ /var/www/html/_$PS_FOLDER_INSTALL/
```

Comment (PierreRambaud) [reply base/config_files/docker_run.sh:96]: Not a good idea to keep the install directory after having a successful installation. Even with a `_`, if someone found the URL, he can reinstall/break your shop :/

This directory is removed only if the installation process is ok, so there is nothing to debug when everything is ok :D

**Review by sowbiba — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @sergioregueira

**Pull request merged**